### DISCUSSION TITLE
Availability of netcdf files to download with goes2go

### DISCUSSION THREAD

**Discussion by gcamposmar** [blaylockbk/goes2go#85] [General]
Hi, I have doubts about if the netcdf files of GOES-16 products that we can download using goes2go library are the same that I can find in the AWS site [*] or through their Python scripts (e.g. download_CMI).

[*] https://home.chpc.utah.edu/~u0553130/Brian_Blaylock/cgi-bin/goes16_download.cgi

It turns out that some files are only available using NOAA-CLASS page (for example, ABI-L2-ACHTF, Cloud Top Temperature 2018-2019) and I can't download with goes2go.

I want to know if exists some web site or repository that mentioned the period that ABI-L2 products are available, or if exist another way (different to CLASS) to access to these files.

**Comment by cmo84** [comment]
These older files are only available through CLASS from what I can tell.  NOAA was only uploading ABI L1b files early in the GOES 16 mission and eventually added the L2 products to the transfer process as well.  Maybe they'd upload them to the bucket if someone asks?  This is the 2nd time this week I've seen people interested in the earlier data.  They seem to be responsive at the email nodd@noaa.gov.  The only issue I can see is that there is an SNS topic that gets populated with any new files and that may mess up some automated processes if they're not expecting a backlog of historical data to be dropped there...
